minio/api.py

```diff
@@ -434,19 +434,18 @@ def remove_all_bucket_notifications(self, bucket_name):
         """
         is_valid_bucket_name(bucket_name)
 
-        content = xml_marshal_bucket_notifications({})
+        content_bytes = xml_marshal_bucket_notifications({})
         headers = {
-            'Content-Length': str(len(content)),
-            'Content-MD5': encode_to_base64(get_md5(content.encode('utf-8')))
```

Comment:
I am using python 3.5 by default for development. It works fine. It also works in python 2.7